frontend/src/features/admin-form/responses/ResponsesPage/storage/UnlockedResponses/ResponsesTable/ResponsesTable.tsx

```diff
@@ -138,6 +238,16 @@ export const ResponsesTable = () => {
     }
   }, [filteredMetadata, metadata, submissionId])
 
+  const columns = useMemo(() => {
```

Comment:
Good to see that this is extracted out to a function rather than choosing to add one more layer of ternary. 👍